### DISCUSSION TITLE
Error While Importing the IEC62056 LIB in python

### DISCUSSION THREAD

**Discussion by ajeshjacob555** [pwitab/iec62056-21#30] [Q&A]
I have installed the IEC62056 module using pip install, still following error is popingout
-------------------------------------------------------------------

from iec62056_21.client import Iec6205621Client

client = Iec6205621Client.with_serial_transport(port='3')

password_challange = client.access_programming_mode()

client.send_password('00000000')  # Common standard password

data_answer = client.read_value('1.8.0')
=======================================================

"C:\Users\Ajesh Jacob\AppData\Local\Microsoft\WindowsApps\python3.9.exe" "C:/Users/Ajesh Jacob/PycharmProjects/pythonProject1/IEC_62056_21.py"
Traceback (most recent call last):
  File "C:\Users\Ajesh Jacob\PycharmProjects\pythonProject1\IEC_62056_21.py", line 1, in <module>
    from iec62056_21.client import Iec6205621Client
ModuleNotFoundError: No module named 'iec62056_21'

Process finished with exit code 1

![image](https://user-images.githubusercontent.com/106435646/170826129-a0b9994b-6242-4731-92ff-1e1535dd190b.png)

**Comment by Krolken** [comment]
Hi.

Im not an expert on python on windows, but it seems like you are not using the correct python environment.

in you file path you have `venv/` part so I am guessing you are using a virtual environment for your project.

But the run-command is running standard python iterpreter which might not have the package install.

Try activate the environment, or change ennvironment in pycharm. You can check if the package is installed with `pip list`

**Reply by h3tz** [reply]
just include the package over settings and include the package there. Right after it will be placed the the right place.
